src/hooks/sdk/useBandwidthMetrics.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useState, useEffect } from 'react';
+import { ClientMetricReport } from 'amazon-chime-sdk-js';
+
+import { useAudioVideo } from '../../providers/AudioVideoProvider';
+
+function isValidMetric(metric: any) {
+  return typeof metric === 'number' && !Number.isNaN(metric);
+}
+
+interface BandwidthMetrics {
+  availableOutgoingBandwidth: number | null;
+  availableIncomingBandwidth: number | null;
+}
+
+export function useBandwidthMetrics() {
+  const audioVideo = useAudioVideo();
+  const [metrics, setMetrics] = useState<BandwidthMetrics>({
+    availableOutgoingBandwidth: null,
+    availableIncomingBandwidth: null,
+  });
+
+  useEffect(() => {
+    if (!audioVideo) {
+      return;
+    }
+
+    const observer = {
+      metricsDidReceive(clientMetricReport: ClientMetricReport): void {
+        const metricReport = clientMetricReport.getObservableMetrics();
+        console.log(metricReport);
```

Comment:
Should we start annotating logs with [DEMO] it would be beneficial to debug if its demo only or [UI-SDK] something similar

Reply:
Oops, I didn't actually mean to include this as a console log. Thanks for catching